### DISCUSSION TITLE
A discord server for developers

### DISCUSSION THREAD

**Discussion by unguest** [roc-streaming/roc-toolkit#337] [Discuss]
I discovered this project 2 days ago and I'm currently working on the implementation of packet jammer functions. I've noticed it would be great to have a discord server to discuss directly (and of course it would be much simpler for newbies like me developing their first implementation ^^).

Maybe there is already something different for that purpose (Slack, Ryver, ...) ?

**Comment by dshil** [comment]
Hi @unguest and thanks for your interest in the project. Currently, we do not use discord. If you want to reach us, you could use one of the [following resources](https://roc-streaming.org/toolkit/docs/about_project/contacts.html).

**Comment by gavv** [comment]
Hi, as mentioned by Dmitriy, we have a mailing list and an IRC channel, feel free to join them. Personally I prefer mailing lists for technical discussions.

IRC is not very convenient because you wont receive messages sent to you when you were offline, but this can be solved by using a Matrix client to login to IRC, as described in our docs. (There are other solutions too).

We use IRC just because it's a kind of de facto standard for open-source projects. Using some modern alternative like discord is OK too if there are enough people who want to use it. Let's keep this issue open and see.

Nowadays some OSS projects are switching from IRC to Matrix. But when choosing between IRC and Matrix, we can probably stay with IRC because you can join IRC channels from Matrix too.

**Comment by gavv** [comment]
I think we can close this for now.

**Comment by gavv** [comment]
Now we also have github discussions.
